### PR TITLE
Bahasa Indonesia translation

### PR DESCRIPTION
**Pull request by rizaumami**
This is Bahasa [Indonesia](https://en.wikipedia.org/wiki/Indonesia) or [Indonesian language (id_ID)](https://en.wikipedia.org/wiki/Indonesian_language) translation for TLPUI.

Feature request: please provide an easy way to switch the languages.

### CODE REVIEW

File: lang/id_ID/LC_MESSAGES/configdescriptions.po [hunk 1]
```diff
@@ -0,0 +1,834 @@
...
+"Pisahkan banyak hosts menggunakan spasi."
+
+#: configschema.json.h:40
+msgid "AHCI_RUNTIME_PM__GROUP_TITLE"
+msgstr "AHCI_RUNTIME_PM"
+
+#: configschema.json.h:41
+msgid "AHCI_RUNTIME_PM__GROUP_DESCRIPTION"
+msgstr ""
+"Runtime Power Management untuk pengenadli AHCI dan diska: on=dimatikan, "
```
Comment (hahn) [inline lang/id_ID/LC_MESSAGES/configdescriptions.po]: pengenadli -> pengendali

Comment (rizaumami) [inline lang/id_ID/LC_MESSAGES/configdescriptions.po]: Oh my ... 
I'll wait this for a while, waiting for another typo to be found

**Comment by d4nj1** [comment]
Any more typos found, or can this be merged?

**Comment by rizaumami** [comment]
@d4nj1 
Yes, you can merge it ([61c9812](https://github.com/d4nj1/TLPUI/pull/19/commits/61c98122818063b6ddc9c62805485210848ff321)).

**Comment by d4nj1** [comment]
Hi, as I changed quite some stuff with in the newest TLPUI version I was not able to merge your code directly. I pushed the translations by myself, but I kept your personal information from the .po-files as the credit goes to you. Thanks again for your support.

**Pull request closed without merging**